### ISSUE TITLE
The Podcast Skipping

### ISSUE THREAD

**Issue by Prakrit-170807** [ModernFlyouts-Community/ModernFlyouts#360]
**Is your feature request related to a problem? Please describe:**
I have been noticing this for a long time this feature i guessed was not a bug because whenever I a listening to a podcast it the controls skips to the next podcast. It would be greate of it only skipped 15 seconds or like 30 seconds or either customise it in the app for a certain duration. 

**Describe the solution you'd like:**
It will not require to go to the streaming app and be able to do it from the controls would be just greate

**Describe alternatives you've considered:**
--NA--

**Additional context:**
--NA--

**Comment by ShankarBUS** [comment]
Well, we recently added a slider to seek the playback position.

What you ask is possible but not a priority for now.

I hope something could done in the future 🙂.

**Comment by Cyberdroid1** [comment]
@Prakrit-170807 
We have recently added the feature of seek bar in the media flyout. Does it satisfy your needs as of now?

**Comment by Prakrit-170807** [comment]
yeah it was nice to use it and it is very helpful. but as short request was that can you pls add a way to turn it off in the settings and an another option to just trigger it with a button as it was before. If you are not willing to add it that is fine. this is a another feature request (pls don't judge me!) can you add a hot key to show the audio control with a activation of the media keys that are play/pause and skipping keys and they interfere with the playing content so just i am saying is that a hot key to show the media controls as that would help to move back and forth with seekbar. hopefully you consider adding these features

**Comment by Prakrit-170807** [comment]
Hello again I just wanted to ask (not acting weird) just wanted to check up on the request I pulled in the previous comment that if they are possible and will you be able to do that if not it is ok or should I open it in a new request also I am worried just because it bother me a lot so hopefully you respond

**Comment by ShankarBUS** [comment]
Bruh,

It could be done! But I am retired!

I can't do anything. @ADeltaX is your only hope.

**Comment by Cyberdroid1** [comment]
Hooking the previous/next buttons on the flyout with the ones of the media app can fix it. Is it possible?

**Comment by Prakrit-170807** [comment]
Thank bro
